src/HIE/Bios/Ghc/Gap.hs

```diff
@@ -21,7 +21,13 @@ module HIE.Bios.Ghc.Gap (
   , unsetLogAction
   ) where
 
-import DynFlags (DynFlags, includePaths)
+import DynFlags (
+    DynFlags
+  , includePaths
+#if __GLASGOW_HASKELL__ >= 810
+  , IncludeSpecs(..)
+#endif
```

Comment:
There is already a conditional import at https://github.com/mpickering/hie-bios/blob/c99d89ccf221ef761af8037bd96170c3756b4a8e/src/HIE/Bios/Ghc/Gap.hs#L56, maybe this can be merged somehow?

Reply:
Good spot, fixed